caikit_nlp/modules/text_embedding/embedding.py

```diff
@@ -670,10 +728,9 @@ def _truncate_input_tokens(
             okay_to_truncate = False
             max_length = max_tokens
 
-        if isinstance(texts[0], str):
-            to_tokenize = [texts]
-        else:
-            assert 0
+        to_tokenize = [texts]
```

Comment:
given this change, I think this line can go after the asserts

Reply:
Agreed. reodering.